README.md

```diff
@@ -496,52 +496,45 @@ docker build -f Dockerfile.armhf -t testarm .
-    $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
-    apt-get update && apt-get install docker-ce -y
-    ```
-
-5. Add user `jenkins` to `docker` group via `usermod -aG docker jenkins`
-6. Login/su as `jenkins`. Since the build process commits to our repos we need to set a global username and email for GitHub:
-
-    ```
-    git config --global user.email "ci@linuxserver.io"
-    git config --global user.name "LinuxServer-CI"
-    ```
-7. Enable buildkit by creating/appending `/etc/docker/daemon.json` with the following:
-    ```
-    {
-      "features": {
-        "buildkit" : true
-      }
-    }
-    ```
-8. *[X86_64 only]* To allow multi-arch builds first you need to register the interpreters with Docker and it needs to be done every time the docker service is re/started. This can be accomplished by creating a new systemd service (ie. `/lib/systemd/system/multiarch.service`) with the following content and enabling it via `sudo systemctl enable multiarch.service`. The command should now run every time the docker service is re/started:
+#### Setting up a Jenkins Build Agent
+
+Jenkins build agents work by being accessible via SSH and having some core programs installed we use for the build process here are the steps to preparing a jenkins build agent:
+
+1. Install docker via the convenience script
+```
+curl -fsSL https://get.docker.com -o get-docker.sh
+sh get-docker.sh
+```
+2. Create a separate build agent container for each executor (customize the paths and the ports)
+```
+services:
+  build-agent-a:
+    image: lscr.io/linuxserver/build-agent:latest
+    privileged: true
+    container_name: build-agent-a
+    environment:
+      - TZ=Etc/UTC
+      - 'PUBLIC_KEY=<insert public key here>'
+    volumes:
+      - /docker-data/build-agent-a:/config
+    ports:
+      - 2222:2222
+    restart: unless-stopped
+  build-agent-b:
+    image: lscr.io/linuxserver/build-agent:latest
+    privileged: true
+    container_name: build-agent-b
+    environment:
+      - TZ=Etc/UTC
+      - 'PUBLIC_KEY=<insert public key here>'
+    volumes:
+      - /docker-data/build-agent-b:/config
+    ports:
+      - 2223:2222
+    restart: unless-stopped
+```
+3. In Jenkins, add each new build agent container as a single executor builder. Set the remote root directory to `/config/jenkins` and don't forget to set the unique port, which is hidden under the advanced button.
+4. *[X86_64 only]* To allow multi-arch builds first you need to register the interpreters with Docker and it needs to be done every time the docker service is re/started. This can be accomplished by creating a new systemd service (ie. `/lib/systemd/system/multiarch.service`) with the following content and enabling it via `sudo systemctl enable multiarch.service`. The command should now run every time the docker service is re/started:
```

Comment:
This only needs to be run on reboot, not docker service restart.

Reply:
tru dat